AsyncDisplayKitTests/ASTextKitTests.mm

```diff
@@ -81,7 +81,7 @@ static BOOL checkAttributes(const ASTextKitAttributes &attributes, const CGSize
   FBSnapshotTestController *controller = [[FBSnapshotTestController alloc] init];
   UIImage *labelImage = UITextViewImageWithAttributes(attributes, constrainedSize, linkTextAttributes);
   UIImage *textKitImage = ASTextKitImageWithAttributes(attributes, constrainedSize);
-  return [controller compareReferenceImage:labelImage toImage:textKitImage error:nil];
+  return [controller compareReferenceImage:labelImage toImage:textKitImage tolerance:0.0 error:nil];
```

Comment:
we're calling a controller method that requires the param. if the tolerance is non-0, it'll track percent diff while scanning pixel values. 0 will make this part of the image diff algorithm a no-op.